_web/_site/about.html

```diff
@@ -265,6 +261,15 @@
 
 
 
+</head>
+
+<body>
+
+
+<div class="container-fluid main-container">
```

Comment:
There seems to be tons of newline spacing; unless this file is automatically generated by a program, we might want to trim the general spacing in the html files.

Reply:
I'd open an issue.